portfolio/src/main/webapp/script.js

```diff
@@ -63,9 +63,9 @@ function four() {
 }
 
 /**
- * Fetches "Hello Leah" message from the server
+ * Fetches messages from the server
  */
-function greeting() {
+function message() {
```

Comment:
Maybe retrieveComments would work better for a name. This is not so much a rule as a style, and it might be pertinent only on Java, but in Java, method names usually start with a verb to indicate that it's doing something.

Message might mean send a message. Better to be explicit for good readability.

Reply:
I just merged it but I'll fix this in the next pull, and I'll keep it in mind from now on